lib/beaker-hostgenerator/data.rb

```diff
@@ -620,6 +620,18 @@ def osinfo
             'packaging_platform' => 'el-7-s390x'
           },
         },
+        'redhat7-AARCH64' => {
+          :general => {
+            'platform'           => 'el-7-aarch64',
+            'packaging_platform' => 'el-7-aarch64'
+          },
+          :abs => {
+            'template' => 'el-7-arm64'
+          },
```

Comment:
@ScottGarman Are the `:abs` entries currently required only for nspooler platforms?

Reply:
@geoffnichols yes, that's my understanding.